docs/_pages/contributing.cn.md

```diff
@@ -88,7 +88,11 @@ and more importantly also try to improve the process along the way as well.
 
  - Once you finalize on a project/task, please open a new JIRA or assign an existing one to yourself. 
       - Almost all PRs should be linked to a JIRA. It's always good to have a JIRA upfront to avoid duplicating efforts.
-      - If the changes are minor, then `[MINOR]` prefix can be added to Pull Request title without a JIRA. 
+      - If the changes are minor, then `[MINOR]` prefix can be added to Pull Request title without a JIRA. There are some tips to help to judge **MINOR** Pull Request :
+        - trivial fixes (for example, a typo, a broken link or an obvious error)
+        - the change is not functionality relevant
+        - changed lines less than 50
```

Comment:
Now I am skeptical of actually quantifying a MINOR PR based on the number of lines. Just by following IDE suggestions, one might modify more than 50-60 lines of code and still it can be considered as a MINOR PR. I guess it is better to not mention number of lines here. Open to hearing others' suggestions.

Reply:
Maybe your opinion is reasonable. The number of lines is only for reference. The main factor should be the content that the PR changed. Let's listen to @vinothchandar 's opinion.

Reply:
We could make the limit larger to say 100.. may be? but I do feel there should be a limit. we are not saying we won't take larger code changes. Just saying if someone is going to change 300 files and cause everyone to rebase/resolve conflicts, lets have them file a JIRA and get some consensus..

Reply:
I was trying to raise a similar point :) I was thinking since anyways this needs mentor's/reviewer's approval, so may be we could do away with this tip here. :) But this is a valid point that mentioning the limit will help make the contributor aware that MINOR PR is supposed to have limited changes. LGTM

Reply:
agree too